specificatie/openapi.yaml

```diff
@@ -1,173 +1,152 @@
 openapi: 3.0.0
 servers:
-  - description: "SwaggerHub API Auto Mocking"
-    url: https://virtserver.swaggerhub.com/VNGRealisatie/api/reisdocumenten
+  - description: |
+                  APILAB testserver
+    url: https://www.haalcentraal.nl/haalcentraal/api/brp
 info:
   title: Reisdocumenten
-  description: "API voor het ontsluiten van gegevens van reisdocumenten uit de GBA en RNI."
-  version: "1.0.0"
+  description: |
+                API voor het ontsluiten van gegevens van reisdocumenten uit de GBA en RNI.
+  version: "0.0.1 (develop)"
   contact:
-    url: https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen
+    url: https://github.com/VNG-Realisatie/Haal-Centraal-Reisdocumenten-bevragen
   license:
     name: European Union Public License, version 1.2 (EUPL-1.2)
     url: https://eupl.eu/1.2/nl/
+tags:
+  - name: Reisdocumenten
 paths:
   /reisdocumenten/{reisdocumentnummer}:
     get:
-      operationId: reisdocumentenReisdocumentnummer
-      description: "Het ophalen van de gegevens van een REISDOCUMENT op basis van het reisdocumentnummer."
+      operationId: GetReisdocument
+      summary: Raadpleeg een reisdocument
+      description: |
+                    Raadpleeg een Reisdocument.
+                    Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, [zie functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-Reisdocumenten-bevragen/blob/v1.0.0/features/fields_extensie.feature).
```

Comment:
ik denk dat voor de meeste gebruikers de fields feature in common interessanter is dan de extensie. Die zegt alleen dat je soms meer krijgt dan gevraagd. kan je ook (eerst) verwijzen naar de standaard fields.feature in common?

Reply:
Ik heb de extensie verwijderd omdat je aangaf dat die hier niet relevant is.